numpy/random/_generator.pyx

```diff
@@ -380,6 +380,20 @@ cdef class Generator:
         out : ndarray or scalar
             Drawn samples from the parameterized exponential distribution.
 
+        Examples
+        --------
+        A real world example: Assume a company has 10000 customer support 
+        agents and each agent spends on average 4 minutes with a 
+        customer.
+        >>> n = 10000
+        >>> z = np.random.default_rng().exponential(scale=4, size=n)
```

Comment:
> Does anyone know why the formatting is so off in CircleCI?

There must be a blank line between the explanatory text and the actual executable example in order for the docstring to parse correctly:

```suggestion
        A real world example: Assume a company has 10000 customer support 
        agents and each agent spends on average 4 minutes with a 
        customer.
        
        >>> n = 10000
        >>> z = np.random.default_rng().exponential(scale=4, size=n)
```

Reply:
Ahh my bad, thank you!

Reply:
That fixed it, thank you!
![image](https://user-images.githubusercontent.com/25092182/190749199-4da4ea0a-c79a-428f-bff9-d17391e4be6f.png)

![image](https://user-images.githubusercontent.com/25092182/190750802-5fb0ac70-d012-43ce-b4c2-f4a5a44c6ea8.png)